### DISCUSSION TITLE
Dumb question - simulating wind/air?

### DISCUSSION THREAD

**Discussion by ckcollab** [InteractiveComputerGraphics/SPlisHSPlasH#204] [Q&A]
First of all: thanks so much for this project, the documentation, and making it free for everyone use. Really appreciate it.

I am trying to make a fun toy example simulating how air moves around a room with a fan. I wonder if this is reasonable with PySplishSplash? In my head I should be able to set some values to 0 (ie surface tension) and get behavior like air moving around a room? May be completely off base, would love guidance.

Thanks, appreciate any help for my dumb questions!

**Comment by janbender** [comment]
I never tried to simulate air but it should be possible. You just have a different rest density and of course you have to adapt some parameters.

**Reply by ckcollab** [reply]
My mistake for making this an issue instead of discussion, thanks for fixing that. I'll dig through the parameters I can find and ask more pointed questions if I have problemos. Glad to hear this will probably work for my little toy project, appreciate it!
